src/components/PresentBar/PresentBar.tsx

```diff
@@ -43,6 +93,24 @@ export function PresentBar() {
           checkedIcon={<PresentToAllIcon />}
         />
       )}
+      {isViewingPresentation && (
+        <>
+          <ToolbarButton
+            aria-label={presentToolsBarPreviousSlide}
+            disabled={isFirstSlideActive}
+            onClick={handleToPreviousSlideClick}
+          >
+            <ChevronLeftIcon />
```

Comment:
The figma design uses `ArrowBackIosIcon` and `ArrowForwardIosIcon`. Is there a reason why it was changed to `Chevron*Icon`?